www/src/Lib/asyncio/__init__.py

```diff
@@ -50,31 +50,31 @@ def process_query():
 FIRST_EXCEPTION = 0
 ALL_COMPLETED = 0
 
-def wait_for(coro_or_future, timeout, *args, loop=None):
-    fut = ensure_future(fut, *args, loop=loop)
+def wait_for(coro_or_future, timeout, *args):
+    fut = ensure_future(fut, *args)
     if timeout:
         def timeout_handler():
             if fut.done():
                 pass
             else:
                 fut.set_exception(TimeoutError())
-        loop = loop or events.get_event_loop()
-        loop.call_later(timeout, timeout_handler)
+        
+        default_event_loop.call_later(timeout, timeout_handler)
     return fut
 
-def ensure_future(coro_or_future, *args, loop=None):
+def ensure_future(coro_or_future, *args):
     if isinstance(coro_or_future, Future):
         return coro_or_future
     elif iscoroutine(coro_or_future):
         return coro_or_future(*args)
     else:
         raise Exception('Expecting coroutine or Future got '+str(coro_or_future)+' instead.')
 
-def gather(*coros_or_futures, loop=None, return_exceptions=False):
+def gather(*coros_or_futures, return_exceptions=False):
     fut_list = [ensure_future(c, loop=loop) for c in coros_or_futures]
```

Comment:
`loop=loop`, but `loop` is no longer an argument.

Shouldn't this refer to the default event loop (defined above in the file) as in `loop=default_event_loop`?